.mergify.yml

```diff
@@ -4,6 +4,7 @@ pull_request_rules:
   - name: automerge to main with label automerge
     conditions:
       - "#approved-reviews-by>=1"
+
```

Comment:
Seems like the formatter does not cover this diff. My bad. Will remove the last two commits.